examples/py_pex_binary/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("//py:defs.bzl", "py_binary", "py_pex_binary")
+
+
+py_binary(
+    name = "binary",
+    srcs = ["say.py"],
+    data = ["data.txt"],
+    env = {
+        "TEST": "1"
+    },
+    deps = [
+        "@pypi_cowsay//:pkg",
+        "@bazel_tools//tools/python/runfiles",
+    ],
+)
+
+
+
```

Comment:
Can these be tidied? I think I've commented on empty space in this diff before, perhaps a pass over would be good.